pwiz/analysis/demux/Jamfile.jam

```diff
@@ -52,9 +52,18 @@ lib pwiz_analysis_demux
         <library>$(PWIZ_ROOT_PATH)/pwiz/data/msdata//pwiz_data_msdata
     ;
 
+lib pwiz_analysis_demuxtestdata
+    :   DemuxTestData.cpp
+    :   <library>$(PWIZ_ROOT_PATH)/pwiz/data/msdata//pwiz_data_msdata
+    :
+    :   <library>$(PWIZ_ROOT_PATH)/pwiz/data/msdata//pwiz_data_msdata
+    ;
+
+
 unit-test-if-exists CubicHermiteSplineTest : CubicHermiteSplineTest.cpp pwiz_analysis_demux ;
 unit-test-if-exists DemuxDebugReadWriteTest : DemuxDebugReadWriteTest.cpp pwiz_analysis_demux ;
 unit-test-if-exists MatrixIOTest : MatrixIOTest.cpp pwiz_analysis_demux ;
+unit-test-if-exists PrecursorMaskCodecTest : PrecursorMaskCodecTest.cpp pwiz_analysis_demux ../../data/msdata//pwiz_data_msdata_examples pwiz_analysis_demuxtestdata ;
```

Comment:
If this test is the only code using `DemuxTestData.cpp`, then you can forgo the lib target and just add it to the source file list (i.e. after `PrecursorMaskCodecTest.cpp`). If you do plan to use it elsewhere, then make it explicit:
`explicit pwiz_analysis_demuxtestdata ;`

Reply:
fixed in 2deaf41